packages/remix-react/links.ts

```diff
@@ -306,11 +306,13 @@ function isHtmlLinkDescriptor(object: any): object is HtmlLinkDescriptor {
   return typeof object.rel === "string" && typeof object.href === "string";
 }
 
-export async function getStylesheetPrefetchLinks(
+export type KeyedHtmlLinkDescriptor = { key: string; link: HtmlLinkDescriptor };
+
+export async function getKeyedPrefetchLinks(
```

Comment:
I renamed this from `getStylesheetPrefetchLinks` to `getKeyedPrefetchLinks` because, apart from now being an array of objects with `key` and `link` so consumers don't need to generate keys, it also makes it clearer that this code also handles any link with `rel` set to `"preload"` (as seen on line `330`).